docs/src/advanced/rendering.md

```diff
@@ -0,0 +1,105 @@
+                "0": "#e5f5f9",
+                "10": "#99d8c9",
+                "255": "#2ca25f",
+            }
+    }
+)
+```
+
+Titiler supports colormaps that are both discrete (where pixels will be one of the colors that you specify) and linear (where pixel colors will blend between the given colors).
+
+For more information, please check out [rio-tiler's docs](https://cogeotiff.github.io/rio-tiler/colormap/).
+
+## Color Formula
+
+Color formulae are simple commands that apply color corrections to images. This is useful for reducing artefacts like atmospheric haze, dark shadows, or muted colors.
+
+Titiler supports color formulae as defined in [Mapbox's `rio-color` plugin](https://github.com/mapbox/rio-color). These include the operations ([taken from the `rio-color` docs](https://github.com/mapbox/rio-color#operations)):
+
+- **Gamma** adjustment adjusts RGB values according to a power law, effectively brightening or darkening the midtones. It can be very effective in satellite imagery for reducing atmospheric haze in the blue and green bands.
+
+- **Sigmoidal** contrast adjustment can alter the contrast and brightness of an image in a way that matches human's non-linear visual perception. It works well to increase contrast without blowing out the very dark shadows or already-bright parts of the image.
+
+- **Saturation** can be thought of as the "colorfulness" of a pixel. Highly saturated colors are intense and almost cartoon-like, low saturation is more muted, closer to black and white. You can adjust saturation independently of brightness and hue but the data must be transformed into a different color space.
+
+In Titiler, color_formulae are applied through the `color_formula` parameter as a string. An example of this option in action:
+
+```python3
+import requests
+
+response = requests.get(
+    f"titiler.xyz/cog/preview",
+    params={
+        "url": "<YOUR COG HERE>",
+        "color_formula": "gamma rg 1.3, sigmoidal rgb 22 0.1, saturation 1.5"
+    }
+)
+```
+
+## Rescaling
+
+Rescaling is the act of adjusting the minimum and maximum values when rendering an image. In an image with a single band, the rescaled minimum value will be set to black, and the rescaled maximum value will be set to white. This is useful if you want to accentuate features that only appear at a certain pixel value (e.g. you have a DEM, but you want to highlight how the terrain changes between sea level and 100m).
+
+Titiler supports rescaling on a per-band basis, using the `rescaling` parameter. The input is a list of comma-delimited min-max ranges (e.g. ["0,100", "100,200", "0,1000]).
+
+```python3
+import requests
+
+response = requests.get(
+    f"titiler.xyz/cog/preview",
+    params={
+        "url": "<YOUR COG HERE>",
+        "rescaling": ["0,100", "0,1000", "0,10000"]
+    }
+)
+```
+
+By default, Titiler will render the maximum value in a band as red (or green, or blue, depending on which band), and the minimum value as black. 
```

Comment:
This is not really true. 

TiTiler/rio-tiler will rescale automatically using min/max values of the input `datatype` (https://github.com/cogeotiff/rio-tiler/blob/dev/rio_tiler/models.py#L699-L723) but this only depends on the output format.